vm/chef/cookbooks/mariadb/files/mariadb-utils

```diff
@@ -0,0 +1,323 @@
+#!/bin/bash
+
+source /opt/c2d/c2d-utils || exit 1
+
+declare -r replication_user="replication"
```

Comment:
imho we shouldn't hide replication user here, better to move it where you declare all users and passwords and pass it here as an argument

Reply:
Done.